drivers/pipes/pipe_common.c

```diff
@@ -772,6 +796,20 @@ int pipecommon_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
         }
         break;
 
+      case PIPEIOC_POLLTHRES:
+        {
+          pipe_ndx_t threshold = (pipe_ndx_t)arg;
```

Comment:
should we support the different threshold for input/output? to avoid the reader and writer interference each other

Reply:
Okay, it will be done.
Current code is okay for our case, but from user's view, it is better to set different threshold for input/output so that it can be used in other places